db/migrate/20171113215700_add_slack_message_channel_field.rb

```diff
@@ -5,6 +5,6 @@ class AddSlackMessageChannelField < ActiveRecord::Migration
   def change
     at = DynamicAnnotation::AnnotationType.where(annotation_type: 'slack_message').last
     ft = DynamicAnnotation::FieldType.where(field_type: 'text').last || create_field_type(field_type: 'text', label: 'Text')
-    create_field_instance annotation_type_object: at, name: 'slack_message_channel', label: 'Slack Message Channel', field_type_object: ft, optional: true
+    create_field_instance annotation_type_object: at, name: 'slack_message_channel', label: 'Slack Message Channel', field_type_object: ft, optional: false
```

Comment:
This mutation was already pushed to `develop`, so changing it won't have effect on existing instances. Please add a new mutation to change the `optional` value for this field instance.